modules/rules_appimage/1.7.4/presubmit.yml

```diff
@@ -0,0 +1,9 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform: [macos, ubuntu2004]
```

Comment:
Can you please specify the supported bazel version in the presubmit.yml? We'll soon enforce https://github.com/bazelbuild/bazel-central-registry/pull/1387

Reply:
Done